### PR TITLE
SAK-30037; make menus work on file helper in select mode

### PR DESCRIPTION
**Pull request by clhedrick**

### CODE REVIEW

File: content/content-tool/tool/src/webapp/vm/content/sakai_filepicker_select.vm [hunk 1]
```diff
@@ -242,7 +262,7 @@ $(document).ready(function() {
 						class="even#if($item.isSelectedForCopy()) selectedCopy#end#if($item.isSelectedForMove()) selectedMove#end#if(!$item.isAvailable()) inactive#end"
 					#end>
 					######################  Item title and action links  ##############
-					<td headers="title" style="text-indent:$width">
+					<td headers="title" style="text-indent:$width" class="xxx">
```
Comment (buckett) [inline content/content-tool/tool/src/webapp/vm/content/sakai_filepicker_select.vm]: I'm not sure this change should be in here, there's no reference to the xxx class.

Comment (clhedrick) [inline content/content-tool/tool/src/webapp/vm/content/sakai_filepicker_select.vm]: Right. A relic of debugging

**Comment by clhedrick** [comment]
I've updated the PR to remove the unnecessary class="xxx" as well as unnecessary changes in spacing.

**Comment by buckett** [comment]
Merging to get the helper working again. Matching the UI in resources would be nice.

**Pull request merged**